backend/src/v1/staging_complaint/staging_complaint.controller.ts

```diff
@@ -14,15 +15,31 @@ import { Public } from "../../auth/decorators/public.decorator";
 export class StagingComplaintController {
   constructor(private readonly stagingComplaintService: StagingComplaintService) {}
 
-  @Post()
+  @Post("/creates")
   @UseGuards(ApiKeyGuard)
-  create(@Body() createStagingComplaint: WebEOCComplaint) {
-    return this.stagingComplaintService.create(createStagingComplaint);
+  createNewComplaint(@Body() createStagingComplaint: WebEOCComplaint) {
+    return this.stagingComplaintService.createNewComplaint(createStagingComplaint);
+  }
+
+  @Post("/updates")
+  @UseGuards(ApiKeyGuard)
+  createComplaintUpdate(@Body() createStagingComplaint: WebEOCComplaintUpdate) {
+    return this.stagingComplaintService.createComplaintUpdate(createStagingComplaint);
   }
 
   @Post("/process/:complaintIdentifier")
   @UseGuards(ApiKeyGuard)
-  async process(@Param("complaintIdentifier") complaintIdentifier: string): Promise<any> {
-    return await this.stagingComplaintService.process(complaintIdentifier);
+  async processWebEOCComplaint(@Param("complaintIdentifier") complaintIdentifier: string): Promise<any> {
+    return await this.stagingComplaintService.processWebEOCComplaint(complaintIdentifier);
+  }
+
+  @Post("/process/:complaintIdentifier/:update_number")
```

Comment:
I know this seems like nitpicking: ```@Post("/process/:complaintIdentifier/:update_number")``` which case_type are you using here, camelCase, or snake_case. should choose one instead of both

Reply:
Good catch.  I've standardized on camel case for parameters.